test/testenv/deployment.go

```diff
@@ -79,6 +87,35 @@ func (d *Deployment) Teardown() error {
 
 	var cleanupErr error
 
+	// Printing Operator pod logs
+	operatorPod := GetOperatorPod(d.testenv.GetTestEnvName())
+	output, _ := exec.Command("kubectl", "logs", "-n", d.testenv.GetTestEnvName(), operatorPod).Output()
+	fmt.Printf("Printing Operator pod logs: \n %v \n", string(output))
+
+	// Saving Splunk pods logs
+	d.testenv.Log.Info("Saving Splunk pods logs")
+	podNames := DumpGetPods(d.testenv.GetTestEnvName())
+	ansibleLog := ":/opt/container_artifact/ansible.log"
+	// Moving current dir to splunk-operator/ to have the logs files saved at correct location:
+	// (splunk-operator/test/test_artifacts/<namespace>/<deployment name>/<podname>.log)
+	currDir, _ := os.Getwd()
+	i := 0
+	for i < 5 {
+		if !strings.HasSuffix(currDir, "splunk-operator") {
+			currDir = filepath.Dir(currDir)
+		}
+		i++
+	}
+	os.Chdir(currDir)
```

Comment:
This is not recommended. It can possibly effect next test running in the same namespace.